src/config.rs

```diff
@@ -14,6 +14,8 @@ pub struct Config {
     pub network_file: String,
     #[arg(long)]
     pub population_file: String,
+    #[arg(long, default_value = "false")]
+    pub adhoc_routing: bool,
```

Comment:
Let's use an enum for this. We have a lot of refactoring in MATSim because people used boolean flags everywhere and then somebody wants to have a third choice. Also something like
```
--routing-mode ad-hoc
```
is more expressive than
```
--adhoc-routing true
```

Reply:
Good point!